benchmarks/auth/README.md

```diff
@@ -0,0 +1,64 @@
+# Auth Benchmark
+
+The auth benchmark ...
```

Comment:
Here is the description of the authorizer where we took the functionality.
Please have a look about the token based autorizer example from AWS